### ISSUE TITLE
no stars?

### ISSUE THREAD

**Issue by lifeiscontent** [alcatraz/Alcatraz#422]
Any chance you'd add a feature to show which plugins have the most starts / most popular?

**Comment by kattrali** [comment]
With a few exceptions, most plugins provide unique functionality. I'm not sure what showing stars would help users to do. Could you elaborate, @lifeiscontent?

**Comment by BeauNouvelle** [comment]
I suppose you could say that about movies or music though. They're different and its all subjective to what people like.

I would personally use stars to sort the plugins as these could be used as an indicator for how useful I may find them.

**Comment by WinterKirk** [comment]
+1 Having some way to determine popular plugins would help someone who is just starting (like me) find some good plugins. It's much like GitHub will show you most starred repos for a given timeframe.

**Comment by writetovibhor** [comment]
Sorting plugins alphabetically makes no sense. I want to see what are popular plugins, don't have time to read through each and every plugin alphabetically.

**Comment by alanhamlett** [comment]
@writetovibhor I think sorting alphabetically makes sense as a default, with ability to sort by popularity.

**Comment by jurre** [comment]
Implementing something like this would require a major infrastructural change, we couldn't just fetch the plugins from a raw JSON file like we do now, instead we'd need to have some sort of backend that polls github for stars regularly and add this data to the existing plugin data.

I'm not sure if this is worth it, given that stars for plugins really don't tell you all that much because of the reasons @kattrali mentioned

**Comment by BeauNouvelle** [comment]
It doesn't even really have to be stars. Perhaps even just rank them by installations.

Of course there's probably still the same issues with infrastructure, but at least you wouldn't be polling.

**Comment by jurre** [comment]
Yeah it's a similarly large overhaul to how we're currently handling things

**Comment by sebastianludwig** [comment]
I came here to see if an issue for this feature already exists or to open one otherwise. At the moment it's impossible to tell which plugins are maintained well and popularity is a good indicator (and motivation) for that. I often open Alcatraz with a thought like "man, xy is annoying, maybe there's a plugin for that". Then I search, open the GitHub pages for all plugins, check the commit activity, number of open issues, stars, etc, last I try out the favorites. Not very streamlined ;-)

I'd find a feature like this _very_ helpful. Wouldn't be a free Heroku instance be enough to periodically (once a day/week?) gather the data and push it into the repo? Or, maybe, Alcatraz should stay the pure package manager, like the `pod` command, and plugin discovery should be solved by a kind-of-separate website like https://cocoapods.org/?

**Comment by jurre** [comment]
I like the idea of a separate website, it would be a lot easier to maintain. Although it would still be an extra burden on the maintainers, since it's another service that needs to be kept up to date.

**Comment by lifeiscontent** [comment]
@jurre as long as there is a way for me to see plugins by popularity it would be helpful.

**Comment by MuhsinFatih** [comment]
+1 I don't see why not. People do want this. On other IDEs and editors I go to popular extensions and usually find very handy extensions that I didn't know would be useful
